index.js

```diff
@@ -19,114 +20,30 @@ let operation = undefined;
 let nr1 = 0;
 let nr2 = 0;
 
-btn1.addEventListener('click', function () {
-    if (operation === undefined) {
-        nr1 = nr1 * 10 + 1;
-        calcScreen.innerHTML = nr1;
-        console.log('nr1', nr1);
-    } else {
-        nr2 = nr2 * 10 + 1;
-        calcScreen.innerHTML = nr2;
-        console.log('nr2', nr2);
-    }
-});
-
-btn2.addEventListener('click', function () {
-    if (operation === undefined) {
-        nr1 = nr1 * 10 + 2;
-        calcScreen.innerHTML = nr1;
-        console.log('nr1', nr1);
-    } else {
-        nr2 = nr2 * 10 + 2;
-        calcScreen.innerHTML = nr2;
-        console.log('nr2', nr2);
-    }
-});
-
-btn3.addEventListener('click', function () {
-    if (operation === undefined) {
-        nr1 = nr1 * 10 + 3;
-        calcScreen.innerHTML = nr1;
-        console.log('nr1', nr1);
-    } else {
-        nr2 = nr2 * 10 + 3;
-        calcScreen.innerHTML = nr2;
-        console.log('nr2', nr2);
-    }
-});
-
-btn4.addEventListener('click', function () {
-    if (operation === undefined) {
-        nr1 = nr1 * 10 + 4;
-        calcScreen.innerHTML = nr1;
-        console.log('nr1', nr1);
-    } else {
-        nr2 = nr2 * 10 + 4;
-        calcScreen.innerHTML = nr2;
-        console.log('nr2', nr2);
-    }
-});
-
-btn5.addEventListener('click', function () {
-    if (operation === undefined) {
-        nr1 = nr1 * 10 + 5;
-        calcScreen.innerHTML = nr1;
-        console.log('nr1', nr1);
-    } else {
-        nr2 = nr2 * 10 + 5;
-        calcScreen.innerHTML = nr2;
-        console.log('nr2', nr2);
-    }
-});
-
-btn6.addEventListener('click', function () {
-    if (operation === undefined) {
-        nr1 = nr1 * 10 + 6;
-        calcScreen.innerHTML = nr1;
-        console.log('nr1', nr1);
-    } else {
-        nr2 = nr2 * 10 + 6;
-        calcScreen.innerHTML = nr2;
-        console.log('nr2', nr2);
-    }
-});
-
-btn7.addEventListener('click', function () {
-    if (operation === undefined) {
-        nr1 = nr1 * 10 + 7;
-        calcScreen.innerHTML = nr1;
-        console.log('nr1', nr1);
-    } else {
-        nr2 = nr2 * 10 + 7;
-        calcScreen.innerHTML = nr2;
-        console.log('nr2', nr2);
-    }
-});
+btn1.addEventListener('click', logButtonNumber);
+btn2.addEventListener('click', logButtonNumber);
+btn3.addEventListener('click', logButtonNumber);
+btn4.addEventListener('click', logButtonNumber);
+btn5.addEventListener('click', logButtonNumber);
+btn6.addEventListener('click', logButtonNumber);
+btn7.addEventListener('click', logButtonNumber);
+btn8.addEventListener('click', logButtonNumber);
+btn9.addEventListener('click', logButtonNumber);
+btn0.addEventListener('click', logButtonNumber);
 
 
-btn8.addEventListener('click', function () {
+function logButtonNumber(event) {
```

Comment:
Rename to `onDigitClick` or `onNumberClick` because the current name makes me think this just logs some data.